docs/sources/configuration/_index.md

```diff
@@ -2334,6 +2337,27 @@ backoff_config:
   [max_retries: <int> | default = 10]
 ```
 
+## index_gateway
+
+The `index_gateway` block configures the Loki index gateway server, responsible for serving index queries
+without the need to constantly interact with the object store.
+
+```yaml
+# Defines in which mode the index gateway server will operate (default to 'simple').
+# It supports two modes:
+# 'simple': an index gateway server instance is responsible for handling,
+#     storing and returning requests for all indices for all tenants.
+# 'ring': an index gateway server instance is responsible for a subset of tenants instead
+#     of all tenants, such that they don't have to be query ready for all existing index
```

Comment:
We don't have to call out query readiness because it is an optional feature and is disabled by default.